### ISSUE TITLE
Update Marker Position

### ISSUE THREAD

**Issue by MartCube** [xon52/x5-gmaps#5]
Hello there, I'm having a task where I need to update user marker position.
I checked the [example code](https://github.com/xon52/x5-gmaps/blob/master/example/ExampleMarkerOptions.vue) provided where you use the **@move** event for marker.
My sample code looks something like this.
`<gmaps-map :options="mapOptions">
		<gmaps-marker :options="shopOptions" @move="updatePosition" />
</gmaps-map>`

`updatePosition(pos) {
	this.shopOptions.position = pos
},`

The code updates the position, however the move event fires too many times and makes the app laggy also trows this vue warning

```
vue.runtime.esm.js?2b0e:619 [Vue warn]: You may have an infinite update loop in watcher with expression "_options"

found in

---> <GmapsMarker> at src/components/Marker.vue
       <GmapsMap> at src/components/Map.vue 
```

Is there some way to get the position **once** when the @move event finish ? 
Or maybe to get the new position when you click on  some "save button".

**Comment by MartCube** [comment]
As a workaround solution I used **$refs** which works nice.
`<gmaps-map :options="mapOptions"> <gmaps-marker :options="shopOptions" ref="shop" /> </gmaps-map>`

then when I click on my save button I run the savePosition() method
`savePosition() { this.shopOptions.position = this.$refs.shop.marker.position.toJSON() }`

If there is some better way to grab position and other props from marker component I would love to know, 
else you can close this issue.

**Comment by xon52** [comment]
I'll have a look at this on the weekend. I actually had to put a guard on some other components (the shapes) to stop this loop. I had better add it to all the other components like the marker too.

Obviously the issue is that Google's position event fires whether the user moves it or whether Vue "moves" it, so you end up in this horrible loop if you assign a new position from the event.

**Comment by xon52** [comment]
This has been fixed in v0.3.14
